brix-core/src/main/java/org/brixcms/plugin/site/SitePlugin.java

```diff
@@ -649,6 +650,12 @@ public SiteTab(IModel<String> title, IModel<Workspace> workspaceModel) {
         public Panel newPanel(String panelId, IModel<Workspace> workspaceModel) {
             return new NodeManagerContainerPanel(panelId, workspaceModel);
         }
+        
+        @Override
+        public boolean isVisible() {
```

Comment:
It is advised to use `#onConfigure()` + `setVisible()` instead of overriding `#isVisible()` for performance reasons.